HST/pipeline/get-program-info.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+##########################################################################################
+# pipeline/get-program-info.py
+#
+# Syntax:
+# get-program-info.py [-h] [--log LOG] [--quiet] proposal_id
+#
+# Enter the --help option to see more information.
+##########################################################################################
+
+import argparse
+import datetime
+import os
+import pdslogger
+import sys
+
+from get_program_info import get_program_info
+from hst_helper import HST_DIR
+
+# Set up parser
+parser = argparse.ArgumentParser(
+    description='get-program-info: Perform mast query with a given proposal id')
+
+parser.add_argument('proposal_id', type=str, default='',
+    help='The proposal id for the mast query')
+
+parser.add_argument('--log', '-l', type=str, default='',
+    help='Path and name for the log file. The name always has the current date and time '+
```

Comment:
You can use """ multi-line strings here. Help ignores extra spaces so indenting the second+ lines doesn't actually show up as spaces in the help message.